demo/Makefile

```diff
@@ -1,9 +1,10 @@
 P= example_als_mcmc example_sgd example_sgd_bpr
 VPATH = ../
-OBJECTS= example_als_mcmc.o example_sgd example_sgd_bpr
-CFLAGS = -std=c99 -g -Wall -O0 -I../include -L../bin -I../externals/CXSparse/Include
-LDLIBS= -lfastfm -L../externals/CXSparse/Lib -lcxsparse -lblas -lm
-CC=gcc
+OBJECTS = example_als_mcmc.o example_sgd example_sgd_bpr
+INCLUDE = -I../include -L../bin -I../externals/CXSparse/Include -I../externals/OpenBLAS
+CFLAGS = -std=c99 -g -Wall -O0 $(INCLUDE)
+LDLIBS = -L../bin -lfastfm -L../externals/CXSparse/Lib -lcxsparse -L../externals/OpenBLAS -lopenblas -lpthread -lm
```

Comment:
`-L../externals/CXSparse/Lib -lcxsparse -L../externals/OpenBLAS -lopenblas -lpthread`
This  libraries should all be included in `-lfastfm`.